### PR TITLE
Add bootstrap-fileinput

### PR DESCRIPTION
**Pull request by kokarn**

### CODE REVIEW

File: files/bootstrap-fileinput/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Kartik Visweswaran"
+github = "https://github.com/kartik-v"
```
Comment (jimaek) [inline files/bootstrap-fileinput/info.ini]: This is the project's repo not the author's
So: `https://github.com/kartik-v/bootstrap-fileinput`

File: files/bootstrap-fileinput/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+homepage = "http://plugins.krajee.com/file-input"
+description = "An enhanced HTML 5 file input for Bootstrap 3.x with file preview, multiple selection, ajax uploads, and more features."
+mainfile = ""
```
Comment (jimaek) [inline files/bootstrap-fileinput/info.ini]: This field is required

File: files/bootstrap-fileinput/update.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "bootstrap-fileinput",
+  "repo": "kartik-v/bootstrap-fileinput",
+  "files": {
+    "include": [ "css/fileinput.min.css", "js/fileinput.min.js" ]
```
Comment (jimaek) [inline files/bootstrap-fileinput/update.json]: Add `./` in front of each path to make sure the bot will create the folders. This way the structure will match your repo

**Comment by kokarn** [comment]
There we go, everything should be up to snuff now

**Comment by jimaek** [comment]
One last thing, can you rename the directory to `bootstrap.fileinput` please?

**Comment by kokarn** [comment]
Absolutely! Should be correct now.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**